torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -314,6 +314,35 @@ void PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
   addGlowNodeValue(outputs[0], output->getResult());
 }
 
+void PyTorchModelLoader::loadLinear(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  assert(inputs.size() == 3);
+  assert(outputs.size() == 1);
+
+  // Indexes of aten::linear inputs
+  struct Inputs {
+	enum {
+	  input = 0,
+	  weights = 1,
+	  bias = 2,
+	}
+  }
+
+  glow::NodeValue input = getGlowNodeValue(inputs[Inputs::input]);
+  glow::NodeValue weights = getGlowNodeValue(inputs[Inputs::weights]);
+  glow::NodeValue bias = getGlowNodeValue(inputs[Inputs::bias]);
```

Comment:
Bias is optional right? We should check `hasGlowNodeValue` first

Reply:
yes it is in linear, although in addmm it is not, but you are right it is better to check since this is linear.